internal/cli/streams/processor/start_test.go

```diff
@@ -0,0 +1,38 @@
+package processor
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/mocks"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestStartOpts_Run(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	mockStore := mocks.NewMockProcessorStarter(ctrl)
+
+	buf := new(bytes.Buffer)
+	startOpts := &StartOpts{
+		store:           mockStore,
+		streamsInstance: "ExampleInstance",
+		processorName:   "ExampleProcessor",
+		OutputOpts: cli.OutputOpts{
+			OutWriter: buf,
+		},
+	}
+
+	mockStore.
+		EXPECT().
+		StartStreamProcessor(gomock.Any()).
```

Comment:
I think we should have stricter assertions on what we pass to the store calls. Splitting the commands out into separate strings might make this easier to assert on then reconstructing Params objects. But it shouldn't be too hard either way.